string.c

```diff
@@ -10450,17 +10450,30 @@ rb_str_unicode_normalized_p(int argc, VALUE *argv, VALUE str)
 /**********************************************************************
  * Document-class: Symbol
  *
- *  <code>Symbol</code> objects represent names and some strings
- *  inside the Ruby
- *  interpreter. They are generated using the <code>:name</code> and
- *  <code>:"string"</code> literals
- *  syntax, and by the various <code>to_sym</code> methods. The same
- *  <code>Symbol</code> object will be created for a given name or string
- *  for the duration of a program's execution, regardless of the context
- *  or meaning of that name. Thus if <code>Fred</code> is a constant in
- *  one context, a method in another, and a class in a third, the
- *  <code>Symbol</code> <code>:Fred</code> will be the same object in
- *  all three contexts.
+ *  <code>Symbol</code> objects are often used like <code>String</code>
+ *  objects to represent names. Symbols are more memory efficient than
```

Comment:
delete sentence: `Symbols are more memory efficient than...`, as you discuss memory characteristics at the end

Reply:
Well, no. That's the point. It's a tease to the longer explanation for people who don't care about the details.

Reply:
But admittedly I could differentiate these two sections or mention there is more detail later in the first one.

Reply:
One thing at a time IMO. You address memory stuff at the end already. Dropping it in here feels disjointed.

Reply:
Okay... ☹️